### ISSUE TITLE
Analyzer IDE0001 not raised in most cases

### ISSUE THREAD

**Issue by 0x084E** [dotnet/roslyn#7491]
Since the Update of Visual Studio to Update 1 the analyzer 'Simply Name' for using directives is not raised anymore in most cases.

Steps to reproduce:
1. Create a new ClassLiabray project (VS default template)
2. Change the Code of Class1.cs to the follwing
```csharp
namespace System.Collections
{
    using System.IO;
    class Class1
    {
        Class1(FileInfo info)
        {
        }
    }
}
```
IDE0001 is raised for System in System.IO.
3. Now copy the file Class1.cs and change the classname in the copied file to Class2.
Now IDE0001 isn't raised anymore in both files. If you delete one file IDE0001 is raised again for the remaining one.

I wrote an analyzer for unqualified using directives in StyleCop.Analyzers (https://github.com/DotNetAnalyzers/StyleCopAnalyzers/pull/1762) which is also not working in such cases. It seems to me that anaylzers for SyntaxKind.UsingDirective aren't even called.

From what I've found out I think this happens if a namespace declaration occurs in multiple files. If you change 'System.Collections' to 'System.Collect' in one of the two files IDE0001 is raised in both.

**Comment by mavasani** [comment]
This is dupe of https://github.com/dotnet/roslyn/issues/7446, which has been fixed post Update1 but has some pending performance investigation - unfortunately, there is no workaround in Update1.

**Comment by 0x084E** [comment]
Thanks for clarification, will wait an update.
